### PR TITLE
fix(DB/GameObject): Add missing Alliance Bonfire in Feralas

### PR DESCRIPTION
**Pull request by Meramaf**
## Changes Proposed:
-  Add missing Spawn to Alliance Bonfire in Feralas.
-  Extinguishing Kalimdor Achivement work now correctly.
- Midsummer Bonfire

## Issues Addressed:

- Closes 

## Tests Performed:

- 
- 

## How to Test the Changes:

1.
2.
3.

## Known Issues and TODO List:

- [ ]
- [ ]

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

**Comment by Maelthyrr** [comment]
@Meramaf why this go  is  alone (21018) ? Do  you have a source for this ?

<details>

WoWScrnShot_062822_081140

</details>

**Comment by Meramaf** [comment]
> @Meramaf why this go  is  alone (21018) ? Do  you have a source for this ?
> 
> 
> 
> <details>
> 
> 
> 
> WoWScrnShot_062822_081140
> 
> 
> 
> </details>

Im ported this from TC. Maybe it for Horde?

**Comment by acidmanifesto** [comment]
https://www.wowhead.com/object=187929/alliance-bonfire
This item is for the midfiresummer fest.
Can we test with the event acitve. there is suppose to be small tent and other stuff near by. we could be missing those as well.

**Comment by Nefertumm** [comment]
21018 is actually from Cataclysm. You can read more about it here https://github.com/azerothcore/azerothcore-wotlk/pull/12129

**Comment by Maelthyrr** [comment]
@Meramaf so can you remove gob 21018? Thanks :)

**Review by acidmanifesto — CHANGES_REQUESTED** [review]
https://github.com/azerothcore/azerothcore-wotlk/pull/12190#issuecomment-1179523173

**Review by acidmanifesto — APPROVED** [review]

File: data/sql/updates/pending_db_world/extinguishing_kalimdor.sql [hunk 1]
```diff
@@ -0,0 +1,6 @@
+DELETE FROM `gameobject` WHERE (`id` = 187929);
+INSERT INTO `gameobject` (`guid`, `id`, `map`, `zoneId`, `areaId`, `spawnMask`, `phaseMask`, `position_x`, `position_y`, `position_z`, `orientation`, `rotation0`, `rotation1`, `rotation2`, `rotation3`, `spawntimesecs`, `animprogress`, `state`, `ScriptName`, `VerifiedBuild`) VALUES
+(164905, 187929, 1, 0, 0, 1, 1, -4412.02, 3480.24, 12.6312, 6.12709, 0, 0, 0.0779684, -0.996956, 300, 0, 1, '', 0);
+
+DELETE FROM `game_event_gameobject` WHERE `eventEntry` = '1' AND `guid` = '242532';
+INSERT INTO `game_event_gameobject` (`eventEntry`, `guid`) VALUES ('1', '164905');
```
Comment (Nefertumm) [inline data/sql/updates/pending_db_world/extinguishing_kalimdor.sql:6]: Don't use ' around integers.

**Review by acidmanifesto — CHANGES_REQUESTED** [review]
Pending Feedback

**Comment by acidmanifesto** [comment]
marking as pending feedback and needs to research to ensure this isnt reverting another pr. that was merged

**Comment by acidmanifesto** [comment]
Do we have any actual verification of this gameobject placement?

**Comment by Maelthyrr** [comment]
> Do we have any actual verification of this gameobject placement?

No until the next event on woltk classic.

**Comment by Kitzunu** [comment]
Closing as there is no reason to have a PR open you need to wait a year on for verification. Open an issue instead

**Pull request closed without merging**